manifests/0000_90_cluster-machine-approver_04_alertrules.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: machineapprover-rules
+  namespace: openshift-cluster-machine-approver
+spec:
+  groups:
+    - name: general.rules
+      rules:
+        - alert: MachineApproverMaxPendingCSRsReached
+          expr: |
+             mapi_current_pending_csr_count > mapi_max_pending_csr_count
```

Comment:
shouldn't this match the names in https://github.com/openshift/cluster-machine-approver/pull/34/files#diff-c47669012d7664f9cbee69c67bf8bedfR11